src/test/java/teammates/test/cases/ui/browsertests/AdminHomePageUiTest.java

```diff
@@ -110,7 +110,7 @@ private void testCreateInstructorAction() throws Exception {
                                         .withRegistrationKey(encryptedKey)
                                         .withInstructorInstitution(institute)
                                         .toAbsoluteString();
-        assertEquals("Instructor AHPUiT Instrúctör has been successfully created with join link:\n" + expectedjoinUrl,
```

Comment:
The `expectedjoinUrl` is not used anymore. Could you delete it? and followed by the `encryptedKey` as well.